plugins/net_plugin/net_plugin.cpp

```diff
@@ -4025,19 +4025,25 @@ namespace eosio {
       case vote_result_t::invalid_signature:
       case vote_result_t::max_exceeded:  // close peer immediately
          fc_elog(vote_logger, "Exceeded max votes per connection for ${c}", ("c", connection_id));
```

Comment:
Not from your PR. But this logging only lists `Exceeded max votes` as the failure reason. It can be unknown_public_key or invalid_signature

Reply:
Changed it.